plane/fit_multi_plane.py

```diff
@@ -1,43 +1,81 @@
 import numpy as np
-from parse_xml import ParseXML
+# from plane.fit_plane import FitPlane
+from plane.parse_xml import ParseXML
+import pandas as pd
 
 class FitMultiPlane:
-    def __init__(self):
+    def __init__(self, fitplanes, real_centers, template_size, um_per_pixel):
```

Comment:
Because this class is becoming closer to coregister_xz's FitPlane, we need to change this calss's name to FitPlane at some point in the future.
Nothing to do right now, just start thinking about it.